e2e/steps_file.js

```diff
@@ -304,7 +304,9 @@ module.exports = function () {
         }
       }
     },
-
+    async getActiveElementIndex() {
```

Comment:
Something to consider with this, we might (but probably won't) run into a situation with multiple collections on a page, at which point it would be useful to be able to pass a collection id from which to pull the remove buttons from. Not something you have to worry about with the changes you have made so I don't mind it being ignored for the time being.